internal/usagegen/usagegen.go

```diff
@@ -86,23 +86,23 @@ func Generate(ctx context.Context, customerID, lang, schemaPath, header, token,
 		return fmt.Errorf("failed to generate usage snippets for %s ✖", lang)
 	}
 
-	if out == "" {
-		if outputBuffer == nil {
+	if outputBuffer == nil {
```

Comment:
This change was required because setting the output directory changed the behaviour of code samples generation to output a file with all the code samples in it which is not needing.

And outputBuffer is only set for code samples generation not the usage snippet generation